znoyder/cli.py

```diff
@@ -16,17 +16,10 @@
 #    under the License.
 #
 
-from argparse import ArgumentParser
-from argparse import Namespace
-from argparse import RawDescriptionHelpFormatter
-from argparse import REMAINDER
-
-from znoyder import browser
-from znoyder import downloader
-from znoyder import finder
-from znoyder import generator
-from znoyder import templater
+from argparse import (REMAINDER, ArgumentParser, Namespace,
+                      RawDescriptionHelpFormatter)
 
+from znoyder import browser, downloader, finder, generator, templater
```

Comment:
Hmmm, AFAIK one import per line is preferred to multiple imports in single line. Why do we change that?

Reply:
That was changed automatically by isort, which is part of the linters used by pre-commit. I used the same config that we have setup in cibyl, I thought I would help to have the same standards, but can be adjusted